src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-editor.component.ts

```diff
@@ -121,6 +149,25 @@ export class ModelingAssessmentEditorComponent implements OnInit {
         });
     }
 
+    /**
+     * Load the feedback suggestions for the current submission from Athena.
+     * @param exercise The current exercise
+     * @param submission The current submission
+     */
+    private async loadFeedbackSuggestions(exercise: ModelingExercise, submission: Submission): Promise<Feedback[]> {
+        try {
+            return (await firstValueFrom(this.athenaService.getModelingFeedbackSuggestions(exercise, submission))) ?? [];
+        } catch (error) {
+            this.alertService.closeAll();
```

Comment:
Why do you close other alerts here?

Reply:
The behavior is the same here as for text and programming exercises - no specific rationale from my side here. I just wanted the UX to be consistent across exercise types. 🤔